data_steward/analytics/cdr_ops/rdr_export_qc.py

```diff
@@ -343,3 +342,46 @@
 AND value_as_string != 'PMI Skip'
 '''
 pd.read_gbq(query, dialect='standard')
+
+# Check pid_rid_mapping table for duplicates
+query = f'''
+SELECT 'person_id' as id_type, person_id as id, COUNT(person_id) as count
+FROM `{project_id}.{new_rdr}.pid_rid_mapping`
+GROUP BY person_id
+HAVING count > 1
+UNION ALL
+SELECT 'research_id' as id_type, research_id as id, COUNT(research_id) as count
+FROM `{project_id}.{new_rdr}.pid_rid_mapping`
+GROUP BY research_id
+HAVING count > 1
+'''
+pd.read_gbq(query, dialect='standard')
+
+# Ensure all person_ids exist in the person table
+query = f'''
+SELECT 'missing_person' as issue_type, person_id
+FROM `{project_id}.{new_rdr}.pid_rid_mapping`
+WHERE person_id NOT IN 
+(SELECT person_id
+FROM `{project_id}.{new_rdr}.person`)
+UNION ALL 
+SELECT 'unmapped_person' as issue_type, person_id
+FROM `{project_id}.{new_rdr}.person`
+WHERE person_id NOT IN 
+(SELECT person_id
+FROM `{project_id}.{new_rdr}.pid_rid_mapping`)
+'''
```

Comment:
I would suggest using the "unofficial coding conventions" found in the rest of the PR.
1. One line for each field selected.  With the 2+ lines starting with the comma-space characters.
2. Adding space around UNION ALL to visually separate the queries.